### DISCUSSION TITLE
photos sync option: idea

### DISCUSSION THREAD

**Discussion by aliasatp** [opengisch/QField#3778] [Ideas]
Hi, is it possible to set an option to avoid photo synchronization to qfieldcloud? My idea is to have the possibility to manage a "big" dataset of photos via traditional way (the so-called cable way) avoiding the upload into the cloud.
In few words an option like: _sync only geodatabase_
Thanks

**Comment by suricactus** [comment]
It is an interesting idea. We are considering to implement sometime in the future such feature either as "do not sync attachments", or as "online preview of QFieldCloud images", or both. Drop us an email info [at] opengis.ch .

**Reply by aliasatp** [reply]
Thanks for your reply.
In my opinion it could be useful to have the possibility to upload photos collected using qfield in the following ways:

**from Qfield side (APP)**
1) as it is now
2) via FTP to a folder of a server (useful if you have a webgis that publish data collected by qfield passing through postgis) avoiding to upload to qfieldcloud--> sync geodatabase and upload photo via FTP
3) simply sync only geodatabase

options 2) and 3) are useful if user syncs data to postgis (to a server)

**from QGIS** is less important because the user can modify (remove, compress...) manually pictures as he/she likes but an automatic option to have preview as you said could be a way to avoid spending time for the user
Thanks again and happy new year
